src/coreclr/inc/clrconfigvalues.h

```diff
@@ -720,7 +720,7 @@ RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableRiscV64Zbb,             W("EnableRiscV64
 #endif
 
 // Runtime-async
-RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_RuntimeAsync, W("RuntimeAsync"), 0, "Enables runtime async method support")
+RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_RuntimeAsync, W("RuntimeAsync"), 1, "Enables runtime async method support")
```

Comment:
I didn't think we were quite ready to have this on by default in previews. For example Thays has some bug fixes to make breakpoints work and that seems like functionality even someone using a preview would expect to have functional.

Reply:
The main reason that prevents us from turning the runtime config switch on is that it breaks other runtime features (e.g. COM interop). Tracked by https://github.com/dotnet/runtime/issues/115094 that needs to be fixed before the runtime config switch is turned on.